### PR TITLE
Structural vehicule controlleur

### PR DESCRIPTION
**Pull request by nibor51**
Just There are only the templates left to do

### CODE REVIEW

File: src/Controller/VehicleController.php [hunk 1]
```diff
@@ -78,4 +80,87 @@ public function delete(Request $request, Vehicle $vehicle, EntityManagerInterfac
...
+        }
+
+        return $this->renderForm('vehicle/parts_new.html.twig', [
+            'vehicle' => $vehicle,
+            'part' => $part,
+            'form' => $form,
+        ]);
+    }
+
+    #[Route('/{id}/parts/{partId}', name: 'vehicle_parts_show', methods: ['GET'])]
```
Comment (rocambille) [inline src/Controller/VehicleController.php]: this is ambiguous: prefer `'/{vehicle}/parts/{part}'`to explicitly connect the first id to vehicle and the second one to part

File: src/Controller/VehicleController.php [hunk 2]
```diff
@@ -78,4 +80,87 @@ public function delete(Request $request, Vehicle $vehicle, EntityManagerInterfac
...
+    public function partsShow(Vehicle $vehicle, Part $part): Response
+    {
+        return $this->render('vehicle/parts_show.html.twig', [
+            'vehicle' => $vehicle,
+            'part' => $part,
+        ]);
+    }
+
+    #[Route('/{id}/parts/{partId}/edit', name: 'vehicle_parts_edit', methods: ['GET', 'POST'])]
```
Comment (rocambille) [inline src/Controller/VehicleController.php]: this is ambiguous: prefer `'/{vehicle}/parts/{part}'`to explicitly connect the first id to vehicle and the second one to part

File: src/Controller/VehicleController.php [hunk 3]
```diff
@@ -78,4 +80,87 @@ public function delete(Request $request, Vehicle $vehicle, EntityManagerInterfac
...
+        }
+
+        return $this->renderForm('vehicle/parts_edit.html.twig', [
+            'vehicle' => $vehicle,
+            'part' => $part,
+            'form' => $form,
+        ]);
+    }
+
+    #[Route('/{id}/parts/{partId}', name: 'vehicle_parts_delete', methods: ['POST'])]
```
Comment (rocambille) [inline src/Controller/VehicleController.php]: this is ambiguous: prefer `'/{vehicle}/parts/{part}'`to explicitly connect the first id to vehicle and the second one to part

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Controller/VehicleController.php (hunk 1 above)
Comment (nibor51) [reply src/Controller/VehicleController.php]: ok I change that for all the methods concerned

**Comment by nibor51** [comment]
Change effective

**Pull request merged**